libgearman-server/plugins/queue/libmemcached/queue.cc

```diff
@@ -313,7 +316,7 @@ static memcached_return callback_loader(const memcached_st*,
   size_t unique_size= strlen(unique);
 
   assert(unique);
-  assert(unique_size);
+  //assert(unique_size);
```

Comment:
If you're going to remove a line from a test, you need to explain why it shouldn't be in the test in the commit message. And don't just comment it out -- remove it. Comments are for clarifying code to humans, and I don't really know what //assert(unique_size); tells humans reading the code.